delta/plugins/elasticsearch/src/main/resources/defaults/default-settings.json

```diff
@@ -1,14 +1,45 @@
 {
   "analysis": {
     "analyzer": {
-      "nexus": {
+      "default": {
         "type": "custom",
         "tokenizer": "classic",
         "filter": [
           "trim",
           "word_delimiter_nexus",
           "lowercase"
         ]
+      },
+      "path_hierarchy": {
+        "filter": [
+          "lowercase"
+        ],
+        "tokenizer": "path_hierarchy",
+        "type": "custom"
+      },
+      "autocomplete": {
+        "type": "custom",
+        "tokenizer": "autocomplete",
+        "filter": [
+          "lowercase"
+        ]
+      },
+      "autocomplete_search": {
+        "tokenizer": "lowercase"
+      }
+    },
+    "tokenizer": {
+      "autocomplete": {
+        "type": "edge_ngram",
+        "min_gram": 2,
+        "max_gram": 10,
```

Comment:
We may have longer words, maybe we should extend that a bit

Reply:
Extended it to 20